saltedge-compliance-connector-java-v2/README.MD

```diff
@@ -12,7 +12,7 @@ The SDK module implements the Salt Edge PSD2 Compliance v2 API compatible with *
    
 ## Application Requirements
 1. JDK, at least version 8 
-1. Spring Boot Framework, at least version 2.6.+
+1. Spring Boot Framework, `>=2.6.4`
```

Comment:
Not sure if this is the correct entry, considering that we have 2.6.4, where are the guarantees that it will work on higher versions?

Reply:
Updated